other/db/data/tables.sql

```diff
@@ -187,14 +187,14 @@ SELECT * FROM bgp_updates WHERE prefix << $1;
 $$ LANGUAGE SQL;
 
 CREATE TABLE IF NOT EXISTS process_states (
-    name VARCHAR (32) UNIQUE,
+    name VARCHAR (63) UNIQUE,
```

Comment:
I will include this in the 23 version migration

Reply:
Fixed in https://github.com/FORTH-ICS-INSPIRE/artemis/pull/472/commits/a2d1863fee24692ee8a84452939512a67cded192